src/Mono.Android/metadata

```diff
@@ -1157,6 +1157,9 @@
   -->
   <remove-node path="/api/package[@name='java.awt.font']/class[@name='NumericShaper']/method[@name='getContextualShaper' and @merge.SourceFile = 'Profiles/api-24.xml.in']" />
   <remove-node path="/api/package[@name='java.awt.font']/class[@name='NumericShaper']/method[@name='getShaper' and @merge.SourceFile = 'Profiles/api-24.xml.in']" />
+  <!-- FIXME: our product had inconsistent metadata fixup duplicates, but the xamarin-android migration from monodroid caused extraneous directory structural changes which broke build to fix this issue. To workaround the problem, these two extraneous metadata fixup lines are added. They should be removed in the future. -->
+  <remove-node path="/api/package[@name='java.awt.font']/class[@name='NumericShaper']/method[@name='getContextualShaper' and @merge.SourceFile = 'api-24.xml.in']" />
```

Comment:
This needs to be `@merge.SourceFile = 'Profiles/api-24.xml.in'`, not `@merge.SourceFile = 'api-24.xml.in'`. Ditto the following line.